templates/footer.twig

```diff
@@ -51,11 +51,18 @@
 						<a href="https://www.openhub.net/p/cakephp">
 							<img alt="OpenHub" src="./resources/open-hub.png">
 						</a>
-						<div class="mt10">
-							<a href="https://www.rackspace.com/">
-								<img alt="Rackspace" src="./resources/rackspace.png">
-							</a>
-						</div>
+                        <div class="mt10">
+                            <a href="https://pingping.io">
+                                {var logoImg = 'resources/pingping.png'}
+                                <img alt="Rackspace" src="{$logoImg|staticFile}">
```

Comment:
This should just be the html. 2.x was re-written and it not a .latte file.

Reply:
Unfortunately I didn't find .html here .twig

Reply:
Yes. The twig has just html. Just use `src="./resources/pingping.png"`.

Reply:
Done, thanks.